docs/source/admin/capacity-planning.md

```diff
@@ -0,0 +1,246 @@
+As long as the total of 8 users _actual_ usage is under 8G, everything is fine.
+But the _limit_ allows a total of 24G to be used,
+which would be a mess if everyone used their full limit.
+But _not_ everyone uses the full limit, which is the point!
+
+This pattern is fine if 1/8 of your users are 'heavy' because _typical_ usage will be ~0.7G,
+and your total usage will be ~5G (1 × 2 + 7 × 0.5 = 5.5).
+
+But if _50%_ of your users are 'heavy' you have a problem because that means your users will be trying to use 10G (4 × 2 + 4 × 0.5 = 10),
+which you don't have.
+
+You can make guesses at these numbers, but the only _real_ way to get them is to measure (more [below](measuring)).
+
+### CPU:memory ratio
+
+Most of the time, you'll find that only one resource is the limiting factor for your users.
+Most often it's memory, but for certain tasks, it could be CPU (or even GPUs).
+
+Many cloud deployments have just one or a few fixed ratios of cpu to memory
+(e.g. 'general purpose', 'high memory', and 'high cpu').
+Setting your secondary resource allocation according to this ratio
+after selecting the more important limit results in a balanced resource allocation.
+
+For instance, some of Google Cloud's ratios are:
+
+| node type   | GB RAM / CPU core |
+| ----------- | ----------------- |
+| n2-highmem  | 8                 |
+| n2-standard | 4                 |
+| n2-highcpu  | 1                 |
+
+### Idleness
+
+Jupyter being an interactive tool means people tend to spend a lot more time reading and thinking than actually running resource-intensive code.
+This significantly affects how much _cpu_ resources a typical active user needs,
+but often does not significantly affect the _memory_.
+
+Ways to think about this:
+
+- More idle users means unused CPU.
+  This generally means setting your CPU _limit_ higher than your CPU _request_.
+- What do your users do when they _are_ running code?
+  Is it typically single-threaded local computation in a notebook?
+  If so, there's little reason to set a limit higher than 1 CPU core.
+- Do typical computations take a long time, or just a few seconds?
+  Longer typical computations means it's more likely for users to be trying to use the CPU at the same moment,
+  suggesting a higher _request_.
+- Even with idle users, parallel computation adds up quickly - one user fully loading 4 cores and 3 using almost nothing still averages to more than a full CPU core per user.
+- Long-running intense computations suggest higher requests.
+
+Again, using mybinder.org as an example—we run around 100 users on 8-core nodes,
+and still see fairly _low_ overall CPU usage on each user node.
+The limit here is actually Kubernetes' pods per node, not memory _or_ CPU.
+This is likely a extreme case, as many Binder users come from clicking links on webpages
+without any actual intention of running code.
+
+![mybinder.org node CPU usage is low with 50-150 users sharing just 8 cores](../images/mybinder-load5.png)
+
```

Comment:
````suggestion
```[figure} ../images/mybinder-load5.png
mybinder.org node CPU usage is low with 50-150 users sharing just 8 cores
```

````